Jbs language styling

jsarabia opened the pull request.

Commit:
Language selection styles and minor tweaks

combobox variables changed to languageSearch
separated the search box from the list view
increased size of magnifying glass in search box

mbr4477 commented on src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/searchablelist/SearchableList.kt:
```diff
@@ -27,12 +27,16 @@ class SearchableList<T>(listItems: ObservableList<T>, outputValue: Property<T>,
         }
+        hbox{
+            addClass(SearchableListStyles.separator)
+        }
```
Instead of adding another UI element as a spacer, it's probably better to just change the spacing between the field and listview directly by setting the searchable list root Vbox spacing to `30.px`

mbr4477 commented on src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectwizard/view/ProjectWizardStyles.kt:
```diff
@@ -121,16 +120,21 @@ class ProjectWizardStyles : Stylesheet() {
             spacing = 10.px
         }
 
         searchableList {
```
Instead of using an hbox with padding to add spacing, try this:
```
searchableList {
    spacing = 30.px
    ...
```

mbr4477 commented on src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/searchablelist/SearchableListStyles.kt:
```diff
@@ -13,11 +16,18 @@ class SearchableListStyles : Stylesheet() {
+        separator {
+            padding = box(15.px)
+            backgroundColor += Color.TRANSPARENT
+        }
```
Can get rid of these styles if using the Vbox `spacing` attribute as mentioned above

Commit:
Replace separator with spacing, move min width

Separator could be replaced with spacing, 
minwith moved to the top before nested elements for readability
